recipes/server_web2.rb

```diff
@@ -50,12 +50,25 @@
 
 # set php time zone
 php_ini = node['platform_family'] == 'rhel' ? '/etc/php.ini' : '/etc/php5/apache2/php.ini'
-ruby_block 'set php timezone' do
-  block do
-    fe = Chef::Util::FileEdit.new(php_ini)
-    fe.search_file_replace_line(/^;date.timezone =.*/, "date.timezone = #{node['time']['timezone']}")
-    fe.write_file
-  end
+execute 'Setup Apache2 PHP5' do
+  command "sudo sed -i -e 's/;date.timezone =/date.timezone = UTC/g' /etc/php5/apache2/php.ini"
+  action :run
+  only_if {File.exists?('/etc/php5/apache2/php.ini')}
+  notifies :reload, 'service[apache2]', :immediately
+end
+
+execute 'Setup Apache2 PHP5.5' do
+  command "sudo sed -i -e 's/;date.timezone =/date.timezone = UTC/g' /etc/php/5.5/apache2/php.ini"
+  action :run
+  only_if {File.exists?('/etc/php/5.5/apache2/php.ini')}
+  notifies :reload, 'service[apache2]', :immediately
+end
+
+execute 'Setup Apache2 PHP' do
+  command "sudo sed -i -e 's/;date.timezone =/date.timezone = UTC/g' /etc/php/5.5/apache2/php.ini"
+  action :run
+  only_if {File.exists?('/etc/php.ini')}
+  notifies :reload, 'service[apache2]', :immediately
 end
```

Comment:
I'm a BASH hound, so naturally I use what I know. That said, I can use the Ruby, and I'll look into both condensing the change down to one block, and to making sure PHP TZ is configurable.